src/test/groovy/graphql/execution/FieldCollectorTest.groovy

```diff
@@ -28,6 +34,8 @@ class FieldCollectorTest extends Specification {
         FieldCollectorParameters fieldCollectorParameters = newParameters()
                 .schema(schema)
                 .objectType(objectType)
+                .instrumentation(SimpleInstrumentation.INSTANCE)
+                .instrumentationState(new CustomInstrumentationState())
```

Comment:
Avoid exception caused by not set `instrumentation`. `instrumentFieldsCollect` will be invoked in this new code.